include/hipSYCL/sycl/libkernel/sscp/group_functions.hpp

```diff
@@ -120,16 +124,8 @@ __acpp_group_broadcast(sub_group g, T x, typename sub_group::linear_id_type loca
   }
 }
 
-template<typename T>
-HIPSYCL_BUILTIN
-T __acpp_group_broadcast(sub_group g, T x,
-                  typename sub_group::id_type local_id) {
-
-  return __acpp_group_broadcast(g, x, static_cast<int>(local_id[0]));
-}
-
-
-template<typename T, int N, class Group>
+template<typename T, int N, class Group,
+          std::enable_if_t<is_group_v<std::decay_t<Group>>, bool> = true>
```

Comment:
We might want to include `group_traits.hpp` for `is_group_v`. Otherwise this will break if the file is ever included in an order that differs from what `sycl.hpp` currently does (e.g., in the stdpar case, we might want to directly include headers without going through the full SYCL header).